prow-jobs/pingcap-tidb-latest-postsubmits.yaml

```diff
@@ -116,6 +116,14 @@ postsubmits:
       skip_report: true
       branches:
         - ^master$
+    - name: pingcap/tidb/merged_integration_typeorm_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: wip/integration-typeorm-test
+      max_concurrency: 1
+      skip_report: true
+      branches:
+        - ^master$
```

Comment:
It is better be a pre-submit type, post submit jobs will just keep build and UT job(for base test coverage uploading)